Masquerade4997Library/src/main/java/Library4997/MasqControlSystems/MasqPID/MasqTurnPIDController.java

```diff
@@ -0,0 +1,107 @@
+package Library4997.MasqControlSystems.MasqPID;
+
+import com.qualcomm.robotcore.util.Range;
```

Comment:
Wtf? Again? The only differences I'm seeing are in the time chance? Why are you changing that? Why tf are there 3 of the same thing.